readme.md

```diff
@@ -168,6 +168,21 @@ function Foo() {
   const [snapshot, state] = useLocalProxy({ count: 0, text: 'hello' })
 ```
 
+### Adding vanilla functions to the state object
+
+For organization' sake, you might want to add functions in your state object. Just be aware that these functions won't have any reactivity, as vanilla Javascript, they work through explicit invocation.
+
```

Comment:
I like this [example](https://twitter.com/0xca0a/status/1329023210768265216) better.

Reply:
I don't see what you mean, in that example the user is assigning to a reactive field the result of a fetch, a Promise.

Reply:
OK I see your point, but I prefer the _addCity_ example in this [video](https://youtu.be/XnlECbwcJZ0?t=461).  I'd rather document Jack Herrington's approach, but both can be documented if you feel strongly about it.

Reply:
I made a small change, but I feel the example should be kept simple and concise.